### ISSUE TITLE
javascript api -- Download button not working

### ISSUE THREAD

**Issue by libjohn** [glin/reactable#237]
I  had a download button working about four months ago.  I've run-run my script and the button no longer works.  I've double checked the documentation and copy/paste/run the [documentation examples](https://glin.github.io/reactable/articles/javascript-api.html#reactable-downloaddatacsv).  They work on the documentation site, but don't work in my environment.  I've tried two different windows environments:

RStudio 2022.02.1+461 "Prairie Trillium" Release (8aaa5d470dd82d615130dbf663ace5c7992d48e3, 2022-03-17) for Windows
Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) QtWebEngine/5.12.8 Chrome/69.0.3497.128 Safari/537.36

& 

RStudio 2022.02.1+461 "Prairie Trillium" Release (8aaa5d470dd82d615130dbf663ace5c7992d48e3, 2022-03-17) for Windows
Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) QtWebEngine/5.12.8 Chrome/69.0.3497.128 Safari/537.36

and tried different browsers.  The documentation example button works fine, but the documentation example scripts are not downloading any CSV data

```{r}
# from https://glin.github.io/reactable/articles/javascript-api.html#reactable-downloaddatacsv
library(htmltools)
library(reactable)

htmltools::browsable(
  tagList(
    tags$button("Download as CSV", onclick = "Reactable.downloadDataCSV('cars-table')"),

    reactable(
      MASS::Cars93[, 1:5],
      defaultPageSize = 5,
      elementId = "cars-table"
    )
  )
)
```

**Comment by glin** [comment]
Could you check what version of reactable you have installed? The JavaScript API is only available in the development version on GitHub (v0.2.3.9000) currently, and not on CRAN yet. If you have v0.2.3.9000 installed, can you open your browser's developer tools console and check if any error messages appear when you click the download button?

**Comment by libjohn** [comment]
Many thanks @glin.  I updated to the development version of reactable and it all works fine.  

I now remember that I originally read the reactable docs where the dev version is documented. THEN, I switch computers and I clearly forgot to document this for myself.  After updating to dev and rerunning my script, it all works.  (I miss my memory!!).

Thanks for your patience and tips.
